templates/interfaces.j2

```diff
@@ -1,49 +1,127 @@
 # vim: sw=4 ts=4 ft=jinja
 # {{ ansible_managed }}
 
+
+{% if network_management_pre_up is defined %}
+post_up {{ network_management_pre_up }}
+{% endif %}
+
 auto lo
 iface lo inet loopback
 
-{% for interface in bridges %}
+{% if network_management_clear_bridges %}
```

Comment:
also check if there are bridges at all.
Otherwise there is a case where OVS is not installed (if there were never bridges on this machine), and this command will fail.

Reply:
not needed, since for will simply skip it's body, when no bridges are defined.